lib/bolt_spec/plans.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
 require 'bolt_spec/plans/mock_executor'
+require 'bolt_spec/event_collector'
```

Comment:
I realize this is currently failing outside of bolt/spec tests. If this approach is approved i think we will want to ship it under `/lib/bolt_spec`?

Reply:
not relevant anymore